app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/produsent/ProdusentRepository.kt

```diff
@@ -208,6 +226,21 @@ class ProdusentRepositoryImpl(
             timestamptz(nyBeskjed.opprettetTidspunkt)
             string(objectMapper.writeValueAsString(nyBeskjed.mottaker))
         }
+        database.nonTransactionalExecuteBatch(
```

Comment:
Tenker umiddelbart at det hadde vært bedre å kjøre disse to i en transaksjon, men det har kanskje ikke så mye å si?